packages/twenty-front/src/modules/object-record/record-filter/utils/applyEmptyFilters.ts

```diff
@@ -0,0 +1,306 @@
+import {
+  ActorFilter,
+  AddressFilter,
+  CurrencyFilter,
+  DateFilter,
+  EmailsFilter,
+  FloatFilter,
+  RecordGqlOperationFilter,
+  RelationFilter,
+  StringFilter,
+  URLFilter,
+  UUIDFilter,
+} from '@/object-record/graphql/types/RecordGqlOperationFilter';
+import {
+  getAddressSubField,
+  getFullNameSubField,
+  getLinkSubField,
+} from '@/object-record/record-filter/utils/getCorrespondingSubfieldFromLabel';
+import { ObjectDropdownFilterDefinition } from '@/object-record/record-filter/utils/turnObjectDropdownFilterIntoQueryFilter';
+import { ViewFilterOperand } from '@/views/types/ViewFilterOperand';
+import { Field } from '~/generated/graphql';
+import { generateILikeFiltersForCompositeFields } from '~/utils/array/generateILikeFiltersForCompositeFields';
+
+export const applyEmptyFilters = (
```

Comment:
- nice code split
- please use pure function as it is more readable (it should return a new object instead of updating the `objectRecordFilters` value)